cluster/gce/util.sh

```diff
@@ -331,6 +331,28 @@ function wait-for-jobs {
   fi
 }
 
+# Robustly try to create a static ip.
+# $1: The name of the ip to create
+function create-static-ip {
+  detect-project
+  local attempt=0
+  while true; do
+    if ! gcloud compute addresses create "$1" \
+      --project "${PROJECT}" \
+      --region "${REGION}" -q > /dev/null; then
```

Comment:
just noticed that `REGION` is declared local on L677, so this will also probably fail